Automattic/wp-calypso
# Docs: Allow `git rebase trunk` or `git merge trunk` in Git workflow

## Proposed Changes

Updates `docs/git-workflow.md` to allow `git rebase trunk` _or_ `git merge trunk`.

The original rationale (7+ years ago) for `git rebase trunk` was:

> While it is tempting to merge from `trunk` into your branch frequently, this leads to a messy history because each merge creates a merge commit.

However, this point is moot with GitHub's "Squash and merge" feature.

An [informal Twitter poll](https://twitter.com/dbchhbr/status/1551891815267151872) sez `git merge trunk` is "the best way to keep your branch up to date", by a narrow margin.

Force-pushing a branch has the unfortunate side effect of making a pull request harder to read. All of the commits end up at the end of the pull request, making it more difficult to understand which comments led to which commits.

## Testing Instructions

1. Try using `git merge trunk` and see how much nicer it is to review a pull request.
